lib/Collection.php

```diff
@@ -241,6 +241,26 @@ public function previousPage($params = null, $opts = null)
         return $this->all($params, $opts);
     }
 
+    /**
+     * Get the first item from the collection.
```

Comment:
```suggestion
     * Gets the first item from the current page. Returns `false` if the current page is empty.
```

Reply:
Would it be better to return `null` here?